### ISSUE TITLE
Integrate logger to test case and util

### ISSUE THREAD

**Issue by shuklanirdesh82** [vmware/docker-volume-vsphere#1525]
golog/logrus or anything else which is really useful to spit out debug/info/warn/error level statements.

**Comment by govint** [comment]
@shuklanirdesh82 this isn't needed, we are already using the log and gocheck has its own logging too.  Where is it needed to create log files of the e2e tests.

**Comment by shuklanirdesh82** [comment]
The issue is meant for handling various log level. Is it supported currently?

**Comment by govint** [comment]
Do we need multiple log levels given that most of the tests have been developed, is there a need to have a debug version of the tests. Preferably logs are always enabled for test code so we have the max. clues for a bug in case it happens.

**Comment by shuklanirdesh82** [comment]
Agree! logger integration is nice to have and we have decided to integrate the logger at some point.

reference: https://github.com/vmware/docker-volume-vsphere/blob/master/tests/utils/esx/esxcli.go#L18

There are many logging statement commented out and waiting for the logger integration. 

One more example I came across: https://github.com/vmware/docker-volume-vsphere/commit/f5837a9ddda07fab220fed6ae71f49eb77b73cbf#diff-0dc9b3cdb9c1477e8377db6251f6e821 (`log.Printf("Error: %s\n", err)`)

IMO, we should avoid using ``log.Printf("Error` ... if we really want such logging then logger integration is a definite need.

/CC @shaominchen
